src/common/fast_align.h

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2015 Glen Joseph Fernandes
+(glenjofe@gmail.com)
```

Comment:
Same as curvefs/src/common/fast_align.h?

Reply:
fix, curvefs/src/common/fast_align.h is removed